docs/consume-packages/PackageDownload-Functionality.md

```diff
@@ -0,0 +1,78 @@
+---
+title: PackageDownload
+description: Describes the PackageDownload feature, which is a complement to PackageReference.
+author: nkolev92
+ms.author: nikolev
+ms.date: 12/22/2021
+ms.topic: conceptual
+---
+
+# PackageDownload
+
+Starting with Visual Studio 2017 and .NET SDK 1.0.0, NuGet [PackageReference](Package-References-in-Project-Files.md) functionality was added.
+
+`PackageReference` allows you to manage your package dependencies directly in your project file.
+When you run restore, the transitive dependencies are resolved automatically and the applicable references are chosen for each package in the project graph.
+
+In [NuGet version 5.3](..\release-notes\NuGet-5.3.md) a companion feature was introduced for [.NET SDK-style projects](..\resources\check-project-format.md) called `PackageDownload`.

```

Comment:
What does the feature do in summary? Would add it after the `PackageDownload` part here. 

i.e. Packages acquired through PackageDownload will undergo the same installation process as packages acquired through PackageReference

Reply:
I did my best to add a one liner. It's pretty hard to describe this feature in one liner. 

tldr is that it downloads a specific version, without its dependencies and without any assets selected. 

Unfortunately that'd be a lot of big words that for which it's not super obvious what they mean, so I tried to simplify it as much as I can.

Reply:
I see you already approved. I'd appreciate a quick look at the addition in this paragraph @JonDouglas

Reply:
looks great! consider this closed.